content/en/tracing/profiling/search_profiles.md

```diff
@@ -0,0 +1,118 @@
+| Allocation Count         | Shows the number of heap allocations made by each function, including allocations which were subsequently freed.     |
+| Wall Time | Shows the elapsed time used by each function. Elapsed time includes time when code is running on CPU, waiting for I/O, and anything else that happens while the function is running. |
+| Exceptions               | Shows the number of caught or uncaught exceptions raised by each function.                                                                                           |
+| Lock                     | Shows the time each function spent in locking (waiting for or holding a lock) or the number of times a function was observed locking/unlocking a lock. |
+| Uncaught Exceptions      | Shows the exceptions that were not caught by any try/except block. |
+| Exceptions              | Shows the exceptions that were raised during program execution. |
+
+{{% /tab %}}
+
+{{% tab "Go" %}}
+
+Once enabled, the following profile types are collected:
+
+| Profile type             | Definition                                                                                                                                                                                                                                                                                         |
+| ------------------------ | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| CPU                      | Shows the time each function spent running on the CPU.                                                                          |
+| Allocation               | Shows the amount of heap memory allocated by each function since the start of the application, including allocations which were subsequently freed. Go calls this `alloc_space`. This is useful for investigating garbage collection load.              |
+| Allocation Count         | Shows the number of objects allocated in heap memory by each function since the start of the application, including allocations which were subsequently freed. This is useful for investigating garbage collection load.     |
+| Heap                     | Shows the amount of heap memory allocated by each function that remained allocated since the start of the application and lived since the last garbage collection. Go calls this `inuse_space`. This is useful for investigating the overall memory usage of your service.               |
+| Heap Count               | Shows the number of objects allocated in heap memory by each function and that remained allocated since the start of the application and lived since the last garbage collection. This is useful for investigating the overall memory usage of your service.                              |
```

Comment:
```suggestion
| Heap Count               | Shows the number of objects allocated in heap memory by each function, and which objects remained allocated since the start of the application and lived since the last garbage collection. This is useful for investigating the overall memory usage of your service.                              |
```